tests/conftest.py

```diff
@@ -15,3 +15,9 @@
 def random_matrix() -> np.ndarray:
     """Generate a random matrix for use in tests."""
     return np.asarray(rng.random((100, 100)))
+
+
+def pytest_configure() -> None:
```

Comment:
This is so cool.
didn't know they have attributes you can import later so easily

Reply:
What is the difference between abs_tolerance and rel_tolerance?

Reply:
Neither did I until recently!